utils/cpu_linux.go

```diff
@@ -260,3 +262,11 @@ func readIntFromFile(filename string) (int, error) {
 
 	return strconv.Atoi(s)
 }
+
+func getPodCPUCount(fs procfs.FS) (int, error) {
```

Comment:
Should this be `getHostCPUCount` as it seems to be getting the total core count for the host?